src/layouts/RootLayout/RootLayout.tsx

```diff
@@ -1,17 +1,17 @@
-'use client';
+"use client";
 
-import React from 'react';
-import { CssBaseline } from '@mui/material';
-import '@fontsource/roboto/300.css';
-import '@fontsource/roboto/400.css';
-import '@fontsource/roboto/500.css';
-import '@fontsource/roboto/700.css';
-import { createTheme, ThemeProvider } from '../../components/dist';
+import React from "react";
+import { CssBaseline } from "@mui/material";
```

Comment:
Blocks also provides a `CssBaseline` and should be favored. However, it may collide with underlying MUI typings. If that is the case, we should get a ticket in the Blocks backlog to address it through either me or Colin